styles.css

```diff
@@ -172,7 +172,7 @@
 
 #page-mod-hsuforum-view #selectgroup label {
     display: block;
-    padding: 0 5px;
+    padding: 0 0.5rem;
```

Comment:
Are you changing the values for padding or you are converting 5px? If you were not changing the values here is the side that can assist with converting `px` to `rem` https://pxtorem.pages.dev/

Reply:
So I can't just change px to rem without changing the value as well, hence it's now 0.5rem. Thanks for the site though